docs/cybersecurity/ddos/process.sql

```diff
@@ -0,0 +1,41 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE STREAM network_traffic (
+  timestamp BIGINT,
+  layers STRUCT<
+   frame STRUCT< 
+      time VARCHAR, 
+      protocols VARCHAR >,
+   eth STRUCT< 
+      src VARCHAR, 
+      dst VARCHAR >,
+   ip STRUCT< 
+      src VARCHAR, 
+      src_host VARCHAR, 
+      dst VARCHAR, 
+      dst_host VARCHAR, 
+      proto VARCHAR >,
+   tcp STRUCT< 
+      srcport VARCHAR, 
+      dstport VARCHAR, 
+      flags_ack VARCHAR, 
+      flags_reset VARCHAR >>
+) WITH (
+  KAFKA_TOPIC = 'network-traffic', 
+  TIMESTAMP = 'timestamp', 
+  VALUE_FORMAT = 'JSON', 
+  PARTITIONS = 6
+);
+
+
+CREATE TABLE potential_slowloris_attacks AS 
+SELECT 
+  layers->ip->src, count(*) as count_connection_reset
+FROM network_traffic 
+  WINDOW TUMBLING (SIZE 60 SECONDS)
+WHERE 
```

Comment:
Other recipes don't have a line break between `WHERE` and the clause, `GROUP BY` and the clause, etc.  For consistency, maybe make them same line.  Separately (in a different PR sweep) we can reevaluate?

So it could be

```
WHERE layers->tcp->flags_ack = '1' AND layers->tcp->flags_reset = '1'
...
etc
```

Reply:
Sure, I think we do have some inconsistency in the existing recipes. I was using customer-360/online-dating/ as the basis. In that "CSAS" you can see they added breaks and whitespace after `HAVING` and `GROUP BY`, and then to have the `WHERE` clause be in-line just felt inconsistent.  The styling is very subjective or course, i'll put the `WHERE` on a single line here.